sdk/core/Azure.Core/src/RequestFailedException.cs

```diff
@@ -61,5 +65,24 @@ public RequestFailedException(int status, string message, string? errorCode, Exc
             Status = status;
             ErrorCode = errorCode;
         }
+
+        /// <inheritdoc />
+        protected RequestFailedException(SerializationInfo info, StreamingContext context)
+            : base(info, context)
```

Comment:
Does `base(...)` serialize the `AdditionalInfo` as well?

Reply:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Exception.cs#L109